cmds/modules/provisiond/main.go

```diff
@@ -111,6 +111,13 @@ func runChecks(ctx context.Context, rootDir string) error {
 
 	log.Error().Str("stderr", buf.String()).Err(err).Msg("integrity check failed, resetting rrd db")
 
+	zui := stubs.NewZUIStub(cl)
+	if er := zui.PushErrors(ctx, "integrity", []string{
+		fmt.Errorf("integrity check failed, resetting rrd db stderr=%s: %w", buf.String(), err).Error(),
```

Comment:
I think you can directly use `fmt.Stringf` instead . You going to lose the error cause anyway once u get the string.